dictionary/README.md

```diff
@@ -1,5 +1,3 @@
-
-Please write a couple of sentences explaining your choice of restart
-strategy.
-
+I felt that the :transient restart option was appropriate for the dictionary because if whatever it is dependent on wants to shutdown normally 
```

Comment:
# -3

The dictionary is a separate application, and so wouldn't (for example) shut down if the hangman game process stops. It only shuts down when the node itself stops, so I'd vote for :permanent.